PoseEstimation-TFLiteSwift/LiveImageViewController.swift

```diff
@@ -13,16 +13,34 @@ class LiveImageViewController: UIViewController {
     
     // MARK: - IBOutlets
     @IBOutlet weak var previewView: UIView?
-    @IBOutlet weak var overlayView: PoseKeypointsDrawingView?
-    var overlayViewRelativeRect: CGRect = .zero
+    @IBOutlet weak var overlayLineDotView: PoseKeypointsDrawingView?
+    @IBOutlet var partButtons: [UIButton]?
+    @IBOutlet weak var thresholdLabel: UILabel?
+    @IBOutlet weak var thresholdSlider: UISlider?
     
-    @IBOutlet weak var thresholdValueLabel: UILabel?
-    @IBOutlet weak var thresholdValueSlider: UISlider?
+    var overlayViewRelativeRect: CGRect = .zero
     
+    lazy var partIndexes: [String: Int] = {
+        var partIndexes: [String: Int] = [:]
+        poseEstimator.partNames.enumerated().forEach { offset, partName in
+            partIndexes[partName] = offset
+        }
+        return partIndexes
+    }()
+    var selectedPartName: String = "ALL"
+    var selectedPartIndex: Int? {
+        guard let partName = selectedPartName.components(separatedBy: "(").first else { return nil }
+        return partIndexes[partName]
```

Comment:
Indice is a plural of index. indexes is not a plural.

Reply:
@MBKwon

Here is similar discuss https://github.com/tucan9389/PoseEstimation-TFLiteSwift/pull/41#discussion_r417717831 

Apple's official document and API use the `indices`, but `indexes` have been also used ([link](https://developer.apple.com/documentation/swift/collection/1641719-indices)). So I think I can also use the `indexes` name. Do you have any opinion about it?

Reply:
I can’t find ‘indexes’ in the link you provide.

Reply:
You can see the discussion in the following link.
- https://github.com/tucan9389/PoseEstimation-TFLiteSwift/pull/41#discussion_r417716157

Here is the screenshot.

![image](https://user-images.githubusercontent.com/37643248/81082596-75739c80-8f2e-11ea-977d-7590b4b1a128.png)

If you cannot find it with the above link, please tell me again.

Thank you.

Reply:
I found about it. You can use both of them. But they have some differences. You choose one you want to use after you see the below. 


https://www.nasdaq.com/articles/indexes-or-indices-whats-the-deal-2016-05-12

Reply:
When I googled, there were two terms of meaning. `Indices` have been used in mathematical(Figure 1), and `Indexes` have been used in publishing(Figure 2). In this context, `Indexes` is more fit but I think there is no precise answer about the subject[[3]](https://stackoverflow.com/a/1379069/4160632). 

So I'll use the `indexes` in this repository. If you have any other opinions, please comment feel free.

| Figure 1. Index of mathematic[[1]](https://images.app.goo.gl/ot6ytTzGaKxsg16x8) |
| ---- |
| <img src="https://user-images.githubusercontent.com/37643248/81464729-1c8e5780-91ff-11ea-8e42-0aeeca007088.png" width=400px> |

| Figure 2. Index of publishing[[2]](https://images.app.goo.gl/a4owkwwa1yQ6gWXA6) |
| ---- | 
| <img src="https://user-images.githubusercontent.com/37643248/81464746-3f207080-91ff-11ea-82dd-6eb3665d5e01.png" width=350px> |

[1] https://images.app.goo.gl/ot6ytTzGaKxsg16x8
[2] https://images.app.goo.gl/a4owkwwa1yQ6gWXA6
[3] https://stackoverflow.com/a/1379069/4160632